package/yast2-network.changes

```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Wed Oct 24 08:26:24 UTC 2018 - mfilka@suse.com
+
+- bnc#1111925
```

Comment:
Not mentioned the new WPA EAP defaults, I would add it

Reply:
ok